docker-entrypoint.sh

```diff
@@ -65,13 +69,15 @@ if [ "$1" = 'ssh-agent' ]; then
     echo '$HOST_GROUP_ID not set, unable to map external and internal GIDs'
     echo 'Using default GID of 1000 instead'
     HOST_GROUP_ID=1000
+    [[ $debug = 'true' ]] && echo "[DEBUG] groupadd -g $HOST_GROUP_ID $HOST_GROUP_NAME"
     groupadd -g $HOST_GROUP_ID $HOST_GROUP_NAME
   else
     if getent group $HOST_GROUP_ID >/dev/null 2>&1; then
-      HOST_GROUP_NAME=$(id -gn $HOST_GROUP_ID)
+      HOST_GROUP_NAME=$(getent group $HOST_GROUP_ID | cut -d: -f1)
```

Comment:
@snw35 This is what was causing the `id: ‘20’: no such user` error. I've tested a local build on OSX, and this will need a bit more testing on the platform(s) you usually test.